client/components/Player.jsx

```diff
@@ -113,12 +129,34 @@ class Player extends React.Component {
 
   pause() {
     clearInterval(this.intervalId);
+    this.resetNowPLaying();
     this.intervalId = null;
   }
 
+  // Methods for finding a comment to display
+  findNowPlaying(newTime) {
+    const showTime = 3;
+    let { nowPlaying } = this.state;
+    const { songTimes } = this.state;
```

Comment:
my guess is that nowPlaying is being changed while songTimes should be a constant - and that's why one has let and the other has const. otherwise  both could be destructerd in one line. right?

Reply:
Yes, I was hoping I could destructure them together, but no such luck! 

```javascript
const { a } = foo; 
let { b } = foo;
```